README.md

```diff
@@ -0,0 +1,85 @@
+
+## How It Works
+1. Zathras is installed to a controller system.
+2. Zathras runs using either a configuration file or by supplying parameters in your command-line interface (CLI) (see the [CLI reference](docs/command_line_reference.md) for more info).  
+3. Test(s) run on the system under test (SUT).
+4. When tests finish, Zathras copies the results back to the controller system.
+5. Once results have been retrieved, Zathras will attempt to cleanup the testing environment (such as by tearing down instances in the cloud or by removing directories and files on bare metal). 
+
+The user configures the Zathras installation with information about the system under test (SUT) and what test(s) to run.
+
+## Getting Started
+
+### Requirements
+To use Zathras, you need: 
+1. A system that will coordinate the execution of tests, which we will call the *control node* or simply the *controller*.
+2. One or more systems where tests will be run. This can be another physical system (aka bare metal), an instance on a public cloud like Amazon Web Services (AWS), or even a virtual machine (VM) either on the controller or another host system.
+
+The following packages will need to be installed on the controller to run Zathras. If you use the included installer script (next section), they will be installed for you.
+- ansible-core
+- git
+- jq
+- python
+- python3-pip
+- terraform
+- yq, version 2.10.0 (newer versions currently break Zathras)
+- wget
+
+### Installation
+Installing Zathras is easy! First, clone this repository to your controller system.
+
+    git clone https://github.com/redhat-performance/zathras.git
+
+Installation can then be completed by running the install script located within the repo's bin/ directory:
+
+    ./install.sh
+
+
+Note: for most installations the install script should be run as a user, not root.
+
+### Test config files
+Test config files, stored in the config/ directory of the Zathras installation, are the connective tissue between Zathras and the test wrappers. Currently this Zathras repo does not contain any defined test config files. Before you can use this tool, you will need to write your own test configs using the included example.yml, or add the configs from an external source if one is available to you.
+
+The process of creating test configuration files is further explained [here](https://github.com/redhat-performance/zathras/blob/documentation-rewrite/docs/test_config_files.md).
+
+### Testing Quickstarts
+In general, testing with Zathras follows a similar set of steps across modalities: configure the tests to run, then run burden via the command line. 
+
+
+However, there are some aspects that are particular to the testing environment being used. Please refer to the following documentation that matches your use case:
+
+- [Testing on bare metal](https://github.com/redhat-performance/zathras/blob/documentation-rewrite/docs/test_config_files.md)
```

Comment:
When you submit, make sure the changes goes to documentation, not documentation-rewrite and you update everything accordingly.